app/src/organisms/ProtocolSetupParameters/ViewOnlyParameters.tsx

```diff
@@ -0,0 +1,136 @@
+import type { SetupScreens } from '../../pages/ProtocolSetup'
+
+export interface ViewOnlyParametersProps {
+  runId: string
+  setSetupScreen: React.Dispatch<React.SetStateAction<SetupScreens>>
+}
+
+export function ViewOnlyParameters({
+  runId,
+  setSetupScreen,
+}: ViewOnlyParametersProps): JSX.Element {
+  const { t, i18n } = useTranslation('protocol_setup')
+  const { makeSnackbar } = useToaster()
+  const mostRecentAnalysis = useMostRecentCompletedAnalysis(runId)
+  const makeSnack = (): void => {
+    makeSnackbar(t('reset_setup'))
+  }
+
+  //  TODO(jr, 3/18/24): remove mockData
+  const parameters = mostRecentAnalysis?.runTimeParameters ?? mockData
+
+  const getDefault = (parameter: RunTimeParameter): string => {
+    const { type, default: defaultValue } = parameter
+    const suffix =
+      'suffix' in parameter && parameter.suffix != null ? parameter.suffix : ''
+    switch (type) {
+      case 'int':
+      case 'float':
+        return `${defaultValue.toString()} ${suffix}`
+      case 'boolean':
+        return Boolean(defaultValue)
+          ? i18n.format(t('on'), 'capitalize')
+          : i18n.format(t('off'), 'capitalize')
+      case 'str':
+        if ('choices' in parameter && parameter.choices != null) {
+          const choice = parameter.choices.find(
+            choice => choice.value === defaultValue
+          )
+          if (choice != null) {
+            return choice.displayName
+          }
+        }
+        break
+    }
+    return ''
+  }
+
+  return (
+    <>
+      <ChildNavigation
+        header={t('parameters')}
+        onClickBack={() => setSetupScreen('prepare to run')}
+        chipProps={{
+          type: 'info',
+          iconName: 'information',
+          text: t('values_are_view_only'),
+        }}
+      />
+      <Flex
+        marginTop="7.75rem"
+        flexDirection={DIRECTION_COLUMN}
+        gridGap={SPACING.spacing8}
+        paddingX={SPACING.spacing8}
+      >
+        <Flex
+          gridGap={SPACING.spacing8}
+          color={COLORS.grey60}
+          fontSize={TYPOGRAPHY.fontSize22}
+          fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+          lineHeight={TYPOGRAPHY.lineHeight28}
+        >
+          <StyledText paddingLeft={SPACING.spacing16} width="50%">
+            {t('name')}
+          </StyledText>
+          <StyledText>{t('value')}</StyledText>
+        </Flex>
+        {parameters.map((parameter, index) => {
+          //  TODO(jr, 3/20/24): plug in the info if the
+          //  parameter changed from the default
+          const hasCustomValue = true
+          return (
+            <Flex
+              onClick={makeSnack}
+              key={`${parameter.displayName}_${index}`}
+              alignItems={ALIGN_CENTER}
+              backgroundColor={COLORS.grey35}
+              borderRadius={BORDERS.borderRadius8}
+              padding={`${SPACING.spacing16} ${SPACING.spacing24}`}
+              gridGap={SPACING.spacing32}
```

Comment:
<img width="486" alt="Screenshot 2024-03-20 at 3 12 48 PM" src="https://github.com/Opentrons/opentrons/assets/474225/27eb430b-cf5c-44b2-87d4-4b59d1ed82ff">

```suggestion
              gridGap={SPACING.spacing24}
```